distributed/tests/test_active_memory_manager.py

```diff
@@ -101,6 +101,65 @@ async def test_auto_start(c, s, a, b):
     assert len(s.tasks["x"].who_has) == 1
 
 
+@gen_cluster(client=True, config=demo_config("drop", key="x"))
+async def test_add_policy(c, s, a, b):
+    p2 = DemoPolicy(action="drop", key="y", n=10, candidates=None)
+    p3 = DemoPolicy(action="drop", key="z", n=10, candidates=None)
+
+    # policies parameter can be:
+    # - None: get from config
+    # - explicit set, which can be empty
+    m1 = s.extensions["amm"]
+    m2 = ActiveMemoryManagerExtension(s, {p2}, register=False, start=False)
+    m3 = ActiveMemoryManagerExtension(s, set(), register=False, start=False)
```

Comment:
Having multiple extensions isn't typical, is it? That's merely for testing, I assume. Maybe I'm wrong but I do not see any tests which use multiple policies. Am I missing anything?

Reply:
The problem is when your global AMM is not started, but you still need some of its functionality.
This whole PR is propaeudetic to the new RetireWorker policy. The problem becomes clear if you look at my WIP branch:
https://github.com/crusaderky/distributed/blob/63737474b8a568ed1d85cf1bea0c969f98f66892/distributed/scheduler.py#L6649-L6681

Reply:
Got it. I was confused for a moment regarding the multiple extensions.

My other point about having multiple policies is obsolete. There are already tests for this